### PR TITLE
Txie

### PR DESCRIPTION
**Pull request by tianchuxie**
add new functions

### CODE REVIEW

File: NewDawn/NewDawn/ProfileViewController/Profile_Height.swift [hunk 1]
```diff
@@ -8,14 +8,100 @@
...
+        if let visible = localReadKeyValue(key: VISIBLE) as? Bool {
+            visibleField = visible
+            // Select the button if a user has already done so
+            if visibleField == true {
+                selectButton(button: visibleButton, text: "Visible")
+            }
+        }
+    }
+    
+    @IBAction func visibleBUttonTapped(_ sender: Any) {
```
Comment (tangziyi001) [inline NewDawn/NewDawn/ProfileViewController/Profile_Height.swift]: visibleButtonTapped?

File: NewDawn/NewDawn/ProfileViewController/Profile_HomeTown.swift [hunk 2]
```diff
@@ -9,14 +9,64 @@
...
+    
+    /* Constant String Keys*/
+    let HOMETOWN = "hometown"
+    let VISIBLE = "hometown_visible"
+    
+    /* Constrains */
+    
+    var visibleField = false
 
+    @IBOutlet weak var homeTwonTextField: UITextField!
```
Comment (tangziyi001) [inline NewDawn/NewDawn/ProfileViewController/Profile_HomeTown.swift]: homeTown

**Comment by tianchuxie** [comment]
screen shot 2019-01-03 at 9 01 26 pm
screen shot 2019-01-03 at 9 01 45 pm
- two UI page completed 
  - height
  -hometown
- image upload incompleted

**Comment by tangziyi001** [comment]
@tianchuxie could you also change the title of this PR

**Pull request merged**